src/fw_nav.c

```diff
@@ -40,6 +43,19 @@ void fw_nav_reset(void)
     }
 }
 
+void fw_FlyTo(void) // PatrikE CruiseMode version
+{
+#define GEO_SKALEFACT    89.832f  // Scale to match meters  
+    int32_t holdHeading = GPS_ground_course / 10;
+    if (holdHeading > 180)
+        holdHeading -= 360;
+    float scaler = ( GEO_SKALEFACT / GPS_scaleLonDown) * cfg.fw_cruise_distance;
+    float wp_lat_diff = cos(holdHeading * 0.0174532925f);
+    float wp_lon_diff = sin(holdHeading * 0.0174532925f) * GPS_scaleLonDown;    
```

Comment:
Remove trailing spaces from the float wp_lon_diff = line (you will do it anyway when you move the variables to top of scope)